qpc/report/tests_report_deployments.py

```diff
@@ -34,6 +34,7 @@
 SUBPARSER = PARSER.add_subparsers(dest="subcommand")
 
 
+# pylint: disable=too-many-public-methods
```

Comment:
no need to disable this directive. it's already disabled in the [global configuration](https://github.com/quipucords/linting/blob/d85e805660ec7e8df2acf6dfe42f88caf4292402/base-flakeheaven.toml#L33)

You might need to clear your pylint cache (mine is located at `~/.cache/pylint`)